### ISSUE TITLE
Feedback on some questions

### ISSUE THREAD

**Issue by Bl-rp** [8values/8values.github.io#80]
Question 2: "It is necessary for the government to intervene in the economy to protect consumers."

When I hear about a government "intervening in the economy", I think specifically of things like corporate subsidies, breaking up conglomerates, controlling the currency etc. I could also see how someone would associate it with corporate regulations like "you can't dump toxic waste in rivers". But as I understand it, this is specifically about consumer protection laws. I think the question is clearer if you word it like "Consumer protection laws are necessary".

Question 3: "The freer the markets, the freer the people."

If I think "the freedom of markets is neither correlated negatively nor positively with the freedom of people", then technically I "disagree" with the statement of the question. On the other hand, it would be natural to think I should pick the middle answer so that the top two are "freer markets => freer people" and the bottom two are "freer markets => less free people". Perhaps this could be solved by a differently worded question but I think the best solution is to add clarification notes to the questions that need it.

Question 6: "International trade is beneficial."

If you don't answer one of the top two, then you think the world would either be as good or better than it currently is if there were zero international trade. That's a rather extreme position to take, and maybe I'm forgetting something but I don't think a more moderate stance on protectionism (for example, mine is "poor countries need protectionism to protect their industries and rich countries may need some protectionism to keep jobs") finds expression in any of the questions in this survey. Perhaps add more questions, or perhaps reword it to "Free international trade is beneficial".

Question 7: "From each according to his ability, to each according to his needs."

This one really needs a note for clarification. I knew this quote is associated with communism, but I used to think "hey, this could be interpreted as 'those who are able to pay more should pay more, and those who need more should get more' - e.g. billionaires should pay more tax than working class people, and disabled people should get government assistance". But then I looked it up on wikipedia and I guess it refers specifically to, as they say, "free access and distribution of goods", meaning "people should pay ALL that they are able, and people should be given ALL they need". The quiz should measure opinion, not knowledge, and how I interpret the question depends on my knowledge of the context of the quote. There should be a note clarifying the meaning of the quote.

Question 8: "It would be best if social programs were abolished in favor of private charity."

What if you think welfare spending should be cut in half by reducing the money given to each program, but not a single program should be completely abolished? Then you'd have to put "disagree". Maybe word it like "reduce government welfare" instead of "abolish social programs".

Question 9: "Taxes should be increased on the rich to provide for the poor."

Same as question 3. Are the green "tax the rich more" and the red "tax the rich less" and the grey "tax the rich the same as now", or should someone who thinks the latter put "disagree" because they technically disagree with the statement? Make a clarification note.

Question 13: "Those with a greater ability to pay should receive better healthcare."

First of all, "willingness to pay" is clearer because "ability to pay" might evoke the idea of a "rich people hospital" and a "poor people hospital" where class may have some part in it. But what does "should receive better healthcare" mean? What if I think there should be public healthcare where everyone is treated equally, but I don't wanna ban private healthcare? The wording "_should_ receive" implies that there _should_ be a high-cost high-quality alternative, and if none springs up organically through market forces, it may be the government's responsibility to provide it - whereas I might think people should be allowed to offer healthcare for money but I'm not bothered if no one does.

Question 21: "It is more important to retain peaceful relations than to further our strength."

Does strength mean military strength? Does "peaceful relations" mean literally peaceful or does it mean amicable? I could see someone hitting "agree" if they think we should be ruthless in pursing the most beneficial trade deals and international agreements and such that we can possibly get except we shouldn't go to war or threaten war for it. Also, something people might not realize when they read the question is that simply building up your military without doing anything directly provocative can be considered something that furthers your strength at the expense of peaceful relations (e.g. Sweden building up its military might anger Russia), so maybe there should be a note to point this out so people don't just think about "aggressive hawkish foreign policy vs peace". The first couple times I took the quiz, I just thought "well we shouldn't go to war or saber-rattle or anything like that".

Question 35: "Government surveillance is necessary in the modern world."

This is a very vague question. If I think the government should be able to wiretap someone they strongly suspect of being a terrorist or international crime lord, does that mean I need to hit the green? I think just about everyone supports that. What the quiz should reflect is the range of opinions from (A) police need specific and probable suspicion in order to get a warrant to surveil someone, to (B) even if there are no specific suspicions, they can surveil certain problem communities (e.g. mosques that are known to produce extremists, or even all (Sunni) mosques, but this can be the difference between "agree" and "strongly agree" of an appropriately worded question), to (C) mass surveillance is a-okay. I think there should be one question asking "is the (B) kind of surveillance okay?" and one question asking "is the (C) kind of surveillance okay?".

Question 42: "Democracy is more than a decision-making process."

What? I have no idea what this means. Needs a clarification note.

Question 64: "Maintaining family values is essential."

Unclear. Can have many different meanings. Split it into different questions, such as "the government shouldn't subsidize single parenthood or enforce child support payments", "divorce should be illegal without a valid reason", "gay adoption should be illegal".

**Comment by brmbrmcar** [comment]
Some I agree on, and many are being discussed already.
